test/api/test_api_helpers.py

```diff
@@ -8,7 +8,7 @@
 import api_helpers
 
 
-def test_escape_dict():
+def test_sanitize_dict():
     messy_dict = {"link": '<a href="http://www.github.com">Click me</a>'}
-    escaped = api_helpers.escape_dict(messy_dict.copy())
-    assert messy_dict['link'] != escaped['link']
+    sanitized = api_helpers.sanitize(messy_dict.copy())
+    assert messy_dict['link'] != sanitized['link']
```

Comment:
here please testing for the actual expected output.

please include a test for dicts

please include a test for objects